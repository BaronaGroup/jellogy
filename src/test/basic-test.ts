import {prepareTestKarhu} from './test-utils'

describe('basic-test', () => {
  const karhuTest = prepareTestKarhu({})

  it('logs things', karhuTest((karhu, output) => {
    karhu.context('test-context').error('Hello, world!')
    expect(output.tracked).toMatchSnapshot()
  }))
})
