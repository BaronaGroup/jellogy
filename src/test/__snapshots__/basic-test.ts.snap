// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`basic-test logs things 1`] = `
Map {
  "NONE" => Array [],
  "DEBUG" => Array [],
  "INFO" => Array [],
  "WARN" => Array [],
  "ERROR" => Array [
    Array [
      "0 ERROR test-context[97m[101m",
      "Hello, world!",
      "[49m[39m",
    ],
  ],
}
`;
